azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/ClientConstants.java

```diff
@@ -166,7 +168,19 @@ private ClientConstants() { }
     static final int DEFAULT_SAS_TOKEN_VALIDITY_IN_SECONDS = 20*60; // 20 minutes
     static final int DEFAULT_SAS_TOKEN_SEND_RETRY_INTERVAL_IN_SECONDS = 5;
     static final String SAS_TOKEN_AUDIENCE_FORMAT = "amqp://%s/%s";
-    
+
+    private static String getClientVersion() {
+        String clientVersion = "";
+        final Properties properties = new Properties();
+        try {
+            properties.load(ClientConstants.class.getResourceAsStream("/client.properties"));
+            clientVersion = properties.getProperty("client.version");
+        } catch (IOException e) {
+        }
```

Comment:
Never ignore this one..May be use some hardcoded value like NOTFOUND if client version is not found as a resource.